report/Final Report/5_InteractionDesign.tex

```diff
@@ -1,14 +1,72 @@
+For this test we used the Thinking-aloud method \citep{thinking-aloud}. This means that we asked our testers to say anything that came up in their mind during the test. This was the easiest way to find out if the testers enjoyed using our product and if the usage was easy enough.\par
+The set-up of the test was based on the way that students of computer science got taught GOAL in the first place. There was a GAMYGDALA manual including; a overview of GAMYGDALA, a list of GAMYGDALA actions in GOAL and a list of the emotion's which could be returned. Besides the manual there was a test environment built on top of the BlockWorld example project of GOAL. The test environment included the full GAMYGDALA plug-in.
+
+\pagebreak
 
 \subsection{Results}
-The results will be discussed here in the final report. This draft does not contain these results yet.
\ No newline at end of file
+The results of the test will be divided into a few different points. We will start with the general reaction on the system. Then we will discuss the positive feedback about the GAMYGDALA plug-in and at last we will discuss the improvements which could be made based on the users' feedback.
+
+\subsubsection{General reaction}
+The overall reaction on the product was very positive. Everyone has emotions and understands basic emotions. Those maybe two of the reason that when the users started reading the overview of GAMYGDALA they were instantly interested. Many of them already started to imagine what it could mean for computer games if GAMYGDALA was applied to it. Because of the interest of the users, it was very easy for us to discuss certain topics of the systems with the testers. They gave very clear and targeted feedback on certain elements of the system. Most of the feedback was positive, it was very clear for users what GAMYGDALA could do and what the emotions were that they got back. The improvement points were mostly based on haziness of variable names, action names or what the variables exactly meant within the engine. The general conclusion was that GAMYGDALA is very cool, but there needs to be more clearity and a basic understanding of the engine to really get the best out of it.
+
+\subsubsection{Positive feedback}
+The positive feedback was generally based on the purpose which GAMYGDALA could have in the (serious) gaming industry. Relations and emotions are really basic elements of life which are not yet fully implemented into (serious) games. Some people directly mentioned the more human taste of non-player characters in-game, if emotions were added to their profile. The GAMYGDALA manual was clear for the users, they did not have questions about what the engine could do. One of the more remarkable reaction about the relation was: 'Oh nice, you can even make a relation with non existing persons'. 
+
+\underline{Millen}
+
+Docs lezen:
```

Comment:
This part were notes from one of the User Interaction Sessions, which contribute to the Interaction Design part of the final report. The notes will be deleted. The pull request now has the label WIP.